### ISSUE TITLE
You need help generating the CovPassCheckApp compatible QR-Code from the partialDcc ?

### ISSUE THREAD

**Issue by dh3wr** [corona-warn-app/cwa-quicktest-onboarding#77]
After long time of investigating, I figured out how to generate from the ``partialDcc`` the corresponding QR-Code, that is shown as valid in the CovPassCheckApp as the last step of https://github.com/corona-warn-app/cwa-quicktest-onboarding/wiki/Anbindung-an-CWA-mit-Verwendung-von-DCCs#verifizierung-der-ausgestellten-zertifikate

If you need support while trying to implement this on your own, contact me at office [a t] ralfwilke.com

BR and happy testing :-)

**Comment by ilya-smirnov-berlin** [comment]
Thanks for your contributions and test certificates examples! We'll get in touch with you soon.

**Comment by henryk** [comment]
Yeah I need help. I mean, no, I can figure it out in less time than it takes you to respond, but that's still more time than if it were documented. This post, saying "I did it, but won't tell you how until you ask me", is strictly worse than https://xkcd.com/979/

https://github.com/eu-digital-green-certificates/dgc-lib/blob/174c63de7caec25be9821af32bee2c84091a2a05/src/test/java/eu/europa/ec/dgc/generation/DgcCryptedPublisherTest.java#L150 has one certificate test case and a reference to `CWA DCC Test Certificates v2.1.docx APPENDIX A` which I couldn't find online, but which probably should be linked in the documentation here.

As for the QR code generation, according to https://github.com/eu-digital-green-certificates/dgc-lib/blob/9a806eb86190c6376fd0e37c5a8e57556d8931ef/src/main/java/eu/europa/ec/dgc/generation/DgcGenerator.java#L114 it's
````
"HC1:" + base45encode(deflate(dccCose))
````
(where `deflate` should be zlib compatible deflate compression at level 9)
